src/core/ext/filters/client_channel/subchannel.cc

```diff
@@ -133,6 +136,8 @@ struct grpc_subchannel {
   bool backoff_begun;
   /** our alarm */
   grpc_timer alarm;
+
+  grpc_core::ChannelTracer* tracer;
```

Comment:
From what i recall, this can be made into a manual constructor. Don't remember what the arguments against were

Reply:
Done